draft-irtf-cfrg-frost.md

```diff
@@ -305,6 +305,12 @@ and
 H2(m) = H(contextString || "chal" || len(m) || m)
 ~~~
 
+and 
+
+~~~
+H3(m) = H(m)
```

Comment:
```suggestion
H3(m) = H(m)

Normally this would also include a domain separator, but for backwards compatibility with other Schnorr signatures, we omit it here.
```